### PR TITLE
Fix tooltip logic

### LINKED ISSUE #256: Fix tooltip logic
Acceptance criteria:

1. Do not show targets in the Totals site when the individual sites do not have targets

Slack thread: https://gnardog.slack.com/archives/C03KGHSSTNC/p1674601054190529?thread_ts=1674585862.272579&cid=C03KGHSSTNC

Image

### PR DESCRIPTION
**Pull request by Elkrival**
closes #256 

Added additional logic to the Totals column to only display counts if the individual sites do not have targets.

Added tests on the tooltip helper functions

### CODE REVIEW

File: views/components/BarGraphTooltip/helpers.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
-import { N_A, TOTAL_LABEL } from '../../../constants'
+import { N_A, TOTAL_LABEL, TOTALS } from '../../../constants'
 
-const formatTooltipData = (studyCounts = 0, studyTargets) =>
+export const formatTooltipData = (studyCounts = 0, studyTargets) =>
   !!studyTargets ? `${studyCounts} / ${studyTargets}` : `${studyCounts}`
 
+export const formatTotalsTooltipData = (
+  { siteCount, siteTarget },
+  displayTargets
```
Comment (mikestone14) [inline views/components/BarGraphTooltip/helpers.js]: is this meant to be a boolean that allows us to determine whether or not to display targets? If so, might be worth naming it `shouldDisplayTarget` to suggest that and passing in a boolean so you don't need the `!!` on the next line.

File: views/components/BarGraphTooltip/helpers.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
-import { N_A, TOTAL_LABEL } from '../../../constants'
+import { N_A, TOTAL_LABEL, TOTALS } from '../../../constants'
 
-const formatTooltipData = (studyCounts = 0, studyTargets) =>
+export const formatTooltipData = (studyCounts = 0, studyTargets) =>
   !!studyTargets ? `${studyCounts} / ${studyTargets}` : `${studyCounts}`
 
+export const formatTotalsTooltipData = (
+  { siteCount, siteTarget },
```
Comment (mikestone14) [inline views/components/BarGraphTooltip/helpers.js]: out of curiosity, why are these in an object and the other parameter is not?

Comment (Elkrival) [reply views/components/BarGraphTooltip/helpers.js]: haha good catch.
https://github.com/AMP-SCZ/dpdash/pull/266/commits/9ef8e4026d6002baf27d9287e747ff5d083b7f50

File: views/components/BarGraphTooltip/helpers.js (hunk 1 above)
Comment (Elkrival) [reply views/components/BarGraphTooltip/helpers.js]: Thanks for the tip.
https://github.com/AMP-SCZ/dpdash/pull/266/commits/bd75abb4ec985aa9d3675d9c9ce18f4c270d3647

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**